tests/softreboot_test.go

```diff
@@ -51,6 +51,17 @@ func WaitForVMIRebooted(vmi *v1.VirtualMachineInstance, login func(vmi *v1.Virtu
 	}, 300)).To(Succeed(), "expected reboot record")
 }
 
+func withoutACPI() libvmi.Option {
```

Comment:
Would it make sense to move that to libvmi/vmi.go? There already a bunch of those there, and that would allow other modules to use it as well.

Reply:
I considered this, but I would personally wait for the second use case in order to "taint" the shared library. Currently, nowhere else in tests do we disable ACPI so I opted out.
If you feel strongly in favor of exporting this NOW, please say so, as I prefer to wait.